app/controllers/template_controller.rb

```diff
@@ -0,0 +1,11 @@
+class TemplateController < ApplicationController
+
+  def index
+  end
+
+  def create
+    logger.debug("TemplateController.create called")
```

Comment:
do we want this `logger.debug` in master?

Reply:
yes - it's currently a debug statement for the Rails logs.  So it should only show up in Rails log when logging level is at :debug.   And chances are we will remove it in the future anyway ... but this keeps it out of logs set to :info.

Reply:
Just a comment, but by default Rails will log this call with:
```
Processing by TemplateController#create as HTML
```